server/webhook.go

```diff
@@ -772,8 +739,16 @@ func (s *Server) createIssueFromPushEvent(ctx context.Context, pushEvent *vcs.Pu
 
 	var migrationInfo *db.MigrationInfo
 	var migrationDetailList []*api.MigrationDetail
-	if repo.Project.SchemaChangeType == api.ProjectSchemaChangeTypeSDL {
-		migrationInfo, migrationDetailList, activityCreateList = s.prepareIssueFromPushEventSDL(ctx, repo, pushEvent, schemaInfo, file, fileType, webhookEndpointID)
+
+	if repo.Project.SchemaChangeType == api.ProjectSchemaChangeTypeDDL && schemaInfo != nil {
+		log.Debug("Ignored schema file for non-SDL", zap.String("file", file), zap.String("type", string(fileType)))
+		return "", false, nil, nil
+	}
+
+	if repo.Project.SchemaChangeType == api.ProjectSchemaChangeTypeSDL && schemaInfo != nil {
+		// Having no schema info indicates that the file is not a schema file (e.g.
+		// "*__LATEST.sql"), try to parse the migration info see if it is a data update.
```

Comment:
This block of comment should be placed in the `else` branch, `schemaInfo != nil` means we _have_ schema info (the opposite).

Reply:
@dragonly - just checking in, how are we doing on the followup for this?

Reply:
It should be the same:
1. Create one issue per schema file using the diff
2. Create as fewer issues as possible for other migration-based files (group by project & database).

Reply:
@dragonly - hmm, not sure I fully understand how these two points relate to this comment block being misplaced during the refactoring?

Reply:
I see. The comments are gone, but a reworded comment is added here.
https://github.com/bytebase/bytebase/blob/cf31ee6f52def4f3439f451a98988a97c8d13661/server/webhook.go#L418

Reply:
Sounds good, thanks!